src/helix-ui/mixins/Positionable.js

```diff
@@ -74,8 +74,9 @@ export const Positionable = (superclass) => {
          * @type {HTMLElement}
          */
         get controlElement () {
-            // FIXME: only run if connected
-            return this.getRootNode().querySelector(`[aria-controls="${this.id}"]`);
+            if (this.getRootNode().isConnected) {
```

Comment:
No need for `getRootNode()` here. We care about if the _instance_ is connected, not its root.

```javascript
if (this.isConnected) {
  ...
}
```

Reply:
😢 Sadly, we'll need to polyfill `isConnected`, because Edge and IE don't support the property.

Reply:
We'll probably want to update the polyfills Guide to reflect that `@webcomponents/webcomponentsjs` adds the polyfill.